.github/workflows/test.yml

```
@@ -0,0 +1,388 @@
+
+      - name: Build application
+        run: pnpm build
+
+      - name: Check build size
+        run: |
+          echo "Build completed successfully"
+          du -sh dist/
+          ls -la dist/
+
+      - name: Upload build artifacts
+        uses: actions/upload-artifact@v3
```

actionlint (1.7.7)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)